source/main.lua

```diff
@@ -262,10 +363,18 @@ else
 end
 
 function playdate.update ()
+    -- refresh screen
+    gfx.sprite.update()
```

Comment:
Problem here is that this wipes out the lose text (you loser or new high score) as that is only displayed once in the addHighScore function. After the player loses there's no need to update the screen so this could just be wrapped in an if statement so it only runs when gameState ~= lost (though there might be a better way to do this).

Reply:
this solution seems simple enough and i can't think of anything else off the top of my head